dist/s_docs

```diff
@@ -205,6 +205,12 @@ EOF
         done
         )
     fi
+
+    # Clean up the doxygen.log if it is empty.
+    doxy_logfile=../src/docs/doxygen.log
+    if [ ! -s "${doxy_logfile}" ] ; then
+        rm -f "${doxy_logfile}"
+    fi
```

Comment:
I'm wondering why not just gitignore the log file if the intention is not showing it with "git diff"?

Reply:
I figured if there's traces in the log file they're worth paying attention to and we're more likely to notice the logs if the file pops up in git diffs. I'm open to gitignoring instead if you'd prefer.

Reply:
Fair enough. I'm fine if that can make the development experience better.